opentelemetry-collector.yaml

```diff
@@ -67,7 +71,7 @@ service:
 
     traces:
       receivers: [otlp]
-      processors: [batch]
+      processors: [attributes/update]
```

Comment:
I'm curious if there is any OTel docs indicating were we found this value? I ask because eventually we'll need to know how to set it up for [this ticket](https://actblue.atlassian.net/browse/DP-439) and I'd love to be able to refer to that doc.

Reply:
here's the documentation! https://docs.honeycomb.io/send-data/opentelemetry/collector/handle-sensitive-information/